starter-code/src/clue.js

```diff
@@ -1,94 +1,127 @@
-// Characters
-/*
-mrGreen
-first_name:   Jacob
-last_name:    Green
-color:        green
-description:  He has a lot of connections
-age:          45
-image:        https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg
-occupation:   Entrepreneur
-
-drOrchid
-first_name:   Doctor
-last_name:    Orchid
-color:        white
-description:  PhD in plant toxicology. Adopted daughter of Mr. Boddy
-age:          26
-image:        http://www.radiotimes.com/uploads/images/Original/111967.jpg
-ocupation:   Scientist
-
-profPlum
-first_name:   Victor
-last_name:    Plum
-color:        purple
-description:  Billionare video game designer
-age:          22
-image:        https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg
-occupation:   Designer
-
-missScarlet
-first_name:   Kasandra
-last_name:    Scarlet
-color:        red
-description:  She is an A-list movie star with a dark past
-age:          31
-image:        https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg
-occupation:   Actor
-
-mrsPeacock
-first_name:   Eleanor
-last_name:    Peacock
-color:        blue
-description:  She is from a wealthy family and uses her status and money to earn popularity
-age:          36
-image:        https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-occupation:   Socialité
-
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-occupation:   Retired Football player
+
+/* 1. Creating the cards */
+
+// characters 
+var mrGreen = {
+first_name:   "Jacob",
+last_name:    "Green",
+color:        "green",
+description:  "He has a lot of connections",
+age:          45,
+image:        "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg",
+occupation:   "Entrepreneur",
+}
+
```

Comment:
Take care to indent your code properly so that it is easily readable.
I can help you find the right package (prettify e.g.) for your text editor to make this super easier for you :)

Reply:
Should look like this:
```
var mrGreen = {
   first_name:    "Jacob", 
   last_name:    "Green", 
   ...
}
```